features/users/user_signup_on_api_sandbox.feature

```diff
@@ -0,0 +1,33 @@
+@javascript @no-seed @focus
+Feature: User signup on api-sandbox
+
+  @on-api-sandbox
+  Scenario: User signs up on the api-sandbox
+    Given I visit "/"
+    Then I should see link 'API Sign up and documentation'
+    And I should see link 'software vendor terms and conditions'
+
+    When I click the link 'API Sign up and documentation'
+    Then I am on the api landing page
+
+    When I click the link 'complete a short registration form'
+    Then I am on the new user sign up page
+    And the page should be accessible
+    And I should see 'Our software vendor terms and conditions contain important information you must agree to'
+
+    When I click the first 'software vendor terms and conditions' link
+    Then I am on the software vendor terms and conditions page
+    And I go back
+
+    When I click govuk checkbox 'I agree to the terms and conditions'
+    Then I fill in 'First name' with 'Jim'
+    And I fill in 'Last name' with 'Bob'
+    And I fill in 'Email' with 'jim.bob@example.com'
+    # # missing field??!
+    # # And I fill in 'Email confirmation' with 'jim.bob@example.com'
```

Comment:
question for @naseberry  - maybe we should be adding this field to sign up?! can you think of any reason why it may not exist already?